### PR TITLE
Fix static date in Dashboard overview with dynamic date range helper method

### LINKED ISSUE #256: DASH: hook up dashboard 'week of' to current date
Dashboard overview has a 'Week of' title that is currently static. Should be hooked up to reflect actual week of the current date. A dashboard helper method seems like one good option.

### PR DESCRIPTION
**Pull request by sco-bo**
* Creates a helper method in `dashboards_helper` to display the current 'week range' in the dashboard overview. 
* Generates a test for the method, which is currently passing

It relates to the following issue #s: 
* Fixes #256

### CODE REVIEW

**Comment by sco-bo** [comment]
Shoot! One situation that I totally didn't take into account is when a particular week spans 2 different months. This PR unfortunately wouldn't address that. Let me know how you want to move forward. Feel free to reject this PR then I can make the necessary changes on my end. Apologies for the error!

File: app/helpers/dashboards_helper.rb [hunk 1]
```diff
@@ -1,2 +1,10 @@
 module DashboardsHelper
+
+  def week_range(date: DateTime.now, start_day: :monday)
```
Comment (colinxfleming) [inline app/helpers/dashboards_helper.rb]: 💯

**Comment by colinxfleming** [comment]
Thanks a ton for the work here @sco-bo ! I'll pull down locally tmrw to confirm behavior, but it looks dead on to me.

**Comment by sco-bo** [comment]
You bet @colinxfleming! Really happy to be able to contribute ;)

File: test/helpers/dashboards_helper_test.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class DashboardsHelperTest < ActionView::TestCase
```
Comment (colinxfleming) [inline test/helpers/dashboards_helper_test.rb]: We should test the helper methods separately in the pregnancies model too! Adding an issue -- not related to this, just making a note that this is a pattern we should follow.

**Comment by colinxfleming** [comment]
Declaring victory and merging. Thanks a bunch @sco-bo !

**Comment by sco-bo** [comment]
Awesome @colinxfleming ! You bet, thanks!

**Pull request merged**